### PR TITLE
[SPARK-37624][PYTHON][DOCS] Suppress warnings for live pandas-on-Spark quickstart notebooks

### PR DESCRIPTION
**Pull request by HyukjinKwon**
### What changes were proposed in this pull request?

This PR proposes to suppress warnings, in live pandas-on-Spark quickstart notebooks, as below:

Screen Shot 2021-12-13 at 2 02 45 PM
Screen Shot 2021-12-13 at 2 02 25 PM
Screen Shot 2021-12-13 at 2 02 20 PM
Screen Shot 2021-12-13 at 1 32 05 PM

### Why are the changes needed?

This is a user-facing quickstart that is interactive shall, and showing a lot of warnings makes difficult to follow the output. Note that we also set a lower log4j level to interpreters. 

### Does this PR introduce _any_ user-facing change?

Users will see clean output in live quickstart notebook: https://mybinder.org/v2/gh/apache/spark/9e614e265f?filepath=python%2Fdocs%2Fsource%2Fgetting_started%2Fquickstart_ps.ipynb

### How was this patch tested?

I manually tested at https://mybinder.org/v2/gh/HyukjinKwon/spark/cleanup-ps-quickstart?labpath=python%2Fdocs%2Fsource%2Fgetting_started%2Fquickstart_ps.ipynb

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
cc @sarutak can you take a quick look please?

**Comment by HyukjinKwon** [comment]
cc @xinrong-databricks too

File: binder/postBuild [hunk 1]
```diff
@@ -33,3 +33,12 @@ else
 fi
 
 pip install plotly "pyspark[sql,ml,mllib,pandas_on_spark]$SPECIFIER$VERSION"
+
+# Set 'PYARROW_IGNORE_TIMEZONE' to surpress warnings from PyArrow.
+echo "export PYARROW_IGNORE_TIMEZONE=1" >> ~/.profile
+
+# Surpress warnings
```
Comment (HyukjinKwon) [inline binder/postBuild:40]:
```suggestion
# Surpress warnings from Spark jobs, and UI progress bar.
```

**Comment by sarutak** [comment]
@HyukjinKwon 
LGTM. BTW, the notebook raises an error which is not relevant to this change (the runtime is 3.2.0, while `pandas_api` is not in this version).
Screenshot from 2021-12-13 16-09-18

So, I applied this change to `branch-3.2` and confirmed it works.
https://hub.gke2.mybinder.org/user/sarutak-spark-wbwn4kx0/doc/workspaces/auto-O/tree/python/docs/source/getting_started/quickstart_ps.ipynb

**Review by xinrong-meng — APPROVED** [review]

**Review by sarutak — APPROVED** [review]

**Comment by xinrong-databricks** [comment]
Hi @sarutak "Binder not found" after clicking the link attached.

May I ask which change you applied?

**Comment by sarutak** [comment]
@xinrong-databricks Oops. Let me check it.

**Comment by sarutak** [comment]
@xinrong-databricks I've modified the URL.

**Comment by xinrong-databricks** [comment]
Got it, thanks @sarutak

**Comment by HyukjinKwon** [comment]
Yeah, I verified it after manually applying and changing it. I should have explained that in the PR description. Thanks for double checking guys.

**Comment by HyukjinKwon** [comment]
Let me merge this in - CI doesn't verify this change.

Merged to master.

**Comment by sarutak** [comment]
Is it better to backport to `branch-3.2` ?

**Comment by HyukjinKwon** [comment]
oh yeah

**Pull request closed without merging**